.github/workflows/azure-deploy.yml

```
@@ -0,0 +1,186 @@
+
+      - name: Azure CLI ログイン
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Docker ログイン (ACR)
+        run: |
+          az acr login --name ${{ env.AZURE_ACR_NAME }}
+
+      - name: Python環境のセットアップ
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)